core/src/main/java/co/aikar/commands/CommandCompletionContext.java

```diff
@@ -101,9 +101,13 @@ public boolean hasConfig(String key) {
                 throw new IllegalStateException("Can not find any parameter that can satisfy " + clazz.getName());
             }
         }
+        return getContextValueByName(clazz, name);
+    }
+
+    public <T> T getContextValueByName(Class<? extends T> clazz, String name) throws InvalidCommandArgument {
```

Comment:
not happy with that. Imo it should be public because somebody might want to use it but without any checks it's a little bit weird. e.g. what happens when no parameter matches this name?

Reply:
I think it's probably fine. there is a check for resolved.containsKey(name) on line 110.